search/stats/stats.md

```diff
@@ -16,6 +16,7 @@ These components are discussed below.
 ### Supported Flags
 
 * `-b`: This flag tells the stats module to always preserve the original body of the entry. For example, saying `tag=* length | stats max(length) | text` would normally cause the output entry's body to be overwritten with the maximum length seen, for convenient display with the text renderer. Adding the `-b` flag (`tag=* length | stats -b max(length) | text`) would preserve the original body, meaning the renderer will instead display the *contents* of the longest entry seen.
+* `-maxtracked <n>`: Sets the maximum number of unique keys to track per operation. This is used to help avoid memory exhaustion if you run `stats count by ipv6addr` and there are millions of IPv6 addresses in the data. Defaults to 100000000.
```

Comment:
should there be an example using the `-maxtracked` flag here too?

Reply:
If you exceed the maxtracked value, the search will terminate with an error.  I should call that out, but since the primary intent of maxtracked is to act as safety rails against bad query choices, I'm not sure it needs an *example*